### DISCUSSION TITLE
Code Blocks

### DISCUSSION THREAD

**Discussion by Drallas** [AppFlowy-IO/AppFlowy#1620] [General]
Are Codeblocks on the Roadmap? 

The current way of displaying code is not appealing to me, and can't find a way to get normal code blocks. The way GitHub does this would be best.

### Appflowy 🤯
<img width="408" alt="Screenshot 2022-12-29 at 19 34 32" src="https://user-images.githubusercontent.com/24792888/209994910-0c6daf15-b9d9-4967-adc3-fa1b902b28ae.png">

### Marktext 🫤
<img width="668" alt="Screenshot 2022-12-29 at 19 34 40" src="https://user-images.githubusercontent.com/24792888/209994912-90657fe0-735e-4baa-8cbe-dbf1f0a902b9.png">

### GitHub 🤩
```python
print("Hello, world!")
for i in range(10):
    print(i)
```

**Comment by LucasXu0** [comment]
Hi, @Drallas. We have supported code block already. You can use the `/` command to insert a code block.

<img width="1127" alt="Screenshot 2022-12-30 at 09 46 14" src="https://user-images.githubusercontent.com/11863087/210026466-d240454a-bc16-449f-b577-68311afa8f27.png">

**Comment by annieappflowy** [comment]
Please let us know your feedback and help us improve this feature @Drallas

**Comment by yyovkov** [comment]
Code Blocks is one of the features which are most used. Is there any way to setup shortcut, so when type three backslashes (```) to open `/code`? This shortcut is very common for the people who are writing markdowns and documentation in notion.

**Comment by annieappflowy** [comment]
Enhanced Code Blocks:
<img width="1016" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/12026239/24b2591a-7510-4ce2-93b0-f7e181c3b8bf">
Please try out with the latest release ([link](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.5.5)).

We'll support ``` in the near future.
Let us know how you like the new Code Block.
